docs/commerce/order_management/configure_order_management.md

```diff
@@ -41,4 +41,10 @@ Then reference it with `ibexa.repositories.<your_repository>.order_management.wo
 
 ``` yaml
 [[= include_file('code_samples/front/shop/order-management/config/packages/ibexa.yaml', 69, 74) =]]
-```
\ No newline at end of file
+```
+
+#### PIM integration
+
+By default, the component integration mechanism reduces product stock values when an order is confirmed and reverts it to the original value when an order is cancelled.
+In your implementation, you may want the reduction/restoration of stock to happen at other stages of the order fulfillment process.
+For this to happen, place the `reduce_stock: true` and/or `restore_stock: true` keys in other places of the workflow.
```

Comment:
It's worth mentioning that each key can be placed only once, so only one stock reduction/restoration.